lib/colorls/flags.rb

```diff
@@ -37,12 +37,16 @@ def process
       # initialize locale from environment
       CLocale.setlocale(CLocale::LC_COLLATE, '')
 
-      return Core.new(@opts).ls if @args.empty?
+      @args = [Dir.pwd] if @args.empty?
       @args.sort!.each_with_index do |path, i|
-        next STDERR.puts "\n   Specified path '#{path}' doesn't exist.".colorize(:red) unless File.exist?(path)
-        puts '' if i > 0
-        puts "#{path}:" if Dir.exist?(path) && @args.size > 1
-        Core.new(path, @opts).ls
+        begin
+          next STDERR.puts "\n   Specified path '#{path}' doesn't exist.".colorize(:red) unless File.exist?(path)
+          puts '' if i > 0
+          puts "\n#{path}:" if Dir.exist?(path) && @args.size > 1
+          Core.new(path, @opts).ls
+        rescue StandardError => e
```

Comment:
Ah, so an error in one directory won't rescue for all directory args. A `directory`-wise rescue of sorts. Good catch! 😄